diff --git a/dist/pull.js b/dist/pull.js
index 9b397bd2e63588dc76c57cd6b752e6363af4df62..166f1bab7a9c7b5d2021b5e72c232ff7397ec8c6 100644
--- a/dist/pull.js
+++ b/dist/pull.js
@@ -183,7 +183,7 @@ function writePage(page, finalMarkdown) {
     ++counts.output_normally;
 }
 const notionLimiter = new limiter_1.RateLimiter({
-    tokensPerInterval: 3,
+    tokensPerInterval: 1,
     interval: "second",
 });
 let notionClient;