diff --git a/node_modules/@sillsdev/docu-notion/dist/transform.js b/node_modules/@sillsdev/docu-notion/dist/transform.js
index 5dc53f7..2cf00d8 100644
--- a/node_modules/@sillsdev/docu-notion/dist/transform.js
+++ b/node_modules/@sillsdev/docu-notion/dist/transform.js
@@ -207,6 +207,21 @@ function getFrontMatter(page) {
     frontmatter += `slug: ${(_a = page.slug) !== null && _a !== void 0 ? _a : ""}\n`;
     if (page.keywords)
         frontmatter += `keywords: [${page.keywords}]\n`;
+    const dateProperty = page.getDateProperty("Date", "", true) || page.getDateProperty("PublishDate", "", true) || "";
+    if (dateProperty) {
+        frontmatter += `date: ${dateProperty}\n`;
+    }
+    const authorsProperty = page.getPlainTextProperty("Authors", "", true) || "";
+    if (authorsProperty) {
+        frontmatter += `authors: [${authorsProperty.split(",").map(author => `"${author.trim()}"`).join(",")}]\n`;
+    }
+    const unlistedProperty =
+    page.metadata?.properties?.Unlisted?.type === "checkbox"
+    ? page.metadata?.properties?.Unlisted?.checkbox
+    : undefined;
+    if (unlistedProperty) {
+        frontmatter += `unlisted: true\n`;
+    }
     frontmatter += "---\n";
     return frontmatter;
 }
