---
layout: nav_menu_item
title: Academic
date: 2011-07-22 21:19:16.000000000 -07:00
type: nav_menu_item
published: true
status: publish
categories: []
tags: []
meta:
  _menu_item_url: ''
  _menu_item_xfn: ''
  _menu_item_classes: a:1:{i:0;s:0:"";}
  _menu_item_target: ''
  _menu_item_type: post_type
  _menu_item_menu_item_parent: '0'
  _menu_item_object_id: '319'
  _menu_item_object: page
author:
  login: dionjwa
  email: dionjw@gmail.com
  display_name: dionjwa
  first_name: Dion
  last_name: Amago
---
<p>[linkedin]: http://www.linkedin.com/in/dionwhiteheadamago<br />
[github]: https://github.com/dionjwa<br />
[ieb]: http://ieb.uni-muenster.de/bioinf<br />
[jedit]: www.jedit.org<br />
[ooo]: http://www.threerings.net</p>
<p>## Dion Whitehead Amago</p>
<p>- Email: dionjw@gmail.com<br />
- Phone: 267-560-7440<br />
- Currently residing in Chicago<br />
- Nationality: New Zealander, US permanent resident.</p>
<p>## Experience</p>
<p>- **2011-Current** Independent software developer.<br />
- **2008-2010** Software developer at [Three Rings Design][ooo].  Designed web-based multiplayer online games.<br />
- **2005-2008** Postdoc fellowship, [Institute of Evolution & Biodiversity][ieb], Münster University, Germany.</p>
<p>## Education</p>
<p>- **2002-2005** PhD in Bioinformatics, [NRW Graduate School](http://www.cebitec.uni-bielefeld.de/gradschool/), Bielefeld University, Germany. Grade: 1.0.<br />
- **2000-2001** Post-graduate Diploma in Genetics, [Massey University](http://www.massey.ac.nz/massey/learning/departments/institute-molecular-biosciences/imbs_home.cfm), New Zealand.<br />
- **1996-2000** B.Sc. Immunology & Virology, [Massey University](http://medlab-school.massey.ac.nz/), New Zealand.</p>
<p>## Skills</p>
<p>- Programming languages, in order of expertise: HaXe, Actionscript, Python, Java, R, Javascript, Bash, Perl.<br />
- Game development, both flash and javascript client, and Java/HaXe server side aspects.<br />
- Modeling and simulations, especially large scale distributed simulations.<br />
- Microarray analysis, mostly with Bioconductor.<br />
- Statistical programming and visualization (using R, Python, and a little Matlab).<br />
- Databases, both SQL and NoSQL, in order of experience: MySql, PostgreSql, Redis.<br />
- Distributed computing (mostly regarding simulations), and cloud computing (Amazon Cloudfront).<br />
- Web development, client and servers-side.<br />
- Regarding bioinformatics, computational biology, and web development, I can probably built it quickly and effectively.<br />
- Languages: English (primary), German (teaching level).<br />
- Version control systems, SVN and Git primarily.</p>
<p>## Teaching</p>
<p>- Bioinformatics I Practicals: Molekulare Evolution und Genomik (German)<br />
- Bioinformatics III (Biocomputing: Python) Lectures  [[1]](content/courses/01.pdf) [[2]](content/courses/02.pdf) [[3]](content/courses/03.pdf) [[4]](content/courses/04.pdf) [[5]](content/courses/05.pdf)<br />
- Introduction to R: Lecture 1 (R basics) [pdf](content/courses/2007.10.17.R-courseI.pdf), Lecture 2 (Statistics up to 2-way ANOVA) [pdf](content/courses/2007.10.17.R-courseII.pdf), Data sets used in R lectures: [Beet.csv](content/courses/Beet.csv), [worms.txt](content/courses/worms.txt)”, [drugs.txt](content/courses/drugs.txt), [drugs2.txt](content/courses/drugs2.txt)<br />
- Introduction to Microarrays (German) [pdf](content/courses/2006.06.12.BioinformatikII.Microarrays.Improved.GERMAN.pdf)</p>
<p>## Research interests</p>
<p>- I love technology, and greatly enjoy programming, especially Haxe and Python.  I am lucky to have my hobby also be my profession.<br />
- I am greatly interested in evolution, as I see this as the most exciting story on the planet, a story with an immense history, and an exciting but uncertain future.<br />
- How gene networks evolve during development, stress response, and in the presence of biochemical noise.<br />
- Microarray analysis, as applied to plant stress response.<br />
- How mutations affect evolution, e.g. gene duplications and phenotypic mutations.<br />
- Visualisation of complex data.</p>
<p>## Publications</p>
<p>- Evolutionary adaption to noise by signalling networks.<br />
D. Whitehead, L. Wissler.<br />
*In revision.*<br />
- [Osmotic and salt stress response in Arabidopsis.](content/courses/Whitehead-Kilian-etal-2007-DRAFT.pdf)<br />
D. Whitehead, J. Kilian, J. Kudla and E. Bornberg-Bauer. *In revision.*<br />
- [The look-ahead effect of phenotypic mutations.](http://www.biology-direct.com/content/3/1/18). D. Whitehead, Claus Wilke, D. Vernazobres<br />
and E. Bornberg-Bauer. *Biology Direct * (2008).<br />
- [The AtGenExpress global stress expression data set: Protocols, evaluation and model data analysis of UV-B light, drought and cold stress responses.](http://www.blackwell-synergy.com/doi/abs/10.1111/j.1365-313X.2007.03052.x) J. Kilian, D. Whitehead, J. Horak, D.<br />
Wanke, S. Weinl, O. Batistic, C. D’Angelo, E. Bornberg-Bauer, J. Kudla and K. Harter. *Plant J* (2007).<br />
- [Chapter 13: Evolution of Regulatory Networks, Systems Biology Applied to Toxicogenomics.](http://www.worldscibooks.com/lifesci/6459.html) Amelie Veron, Dion Whitehead, et. el François Képès (ed.). *World Scientific* (2007).<br />
- [Reconstructing gene function and gene regulatory networks in prokaryotes.](http://bieson.ub.uni-bielefeld.de/volltexte/2005/792) Dion Whitehead. *PhD Thesis* (2005).<br />
- [Handbook of Toxicogenomics: Strategies and Applications 13: Systems Biology Applied to Toxicogenomics.](http://www3.interscience.wiley.com/cgi-bin/abstract/110575170/ABSTRACT) Klaus Prank, Matthias Hoechsmann, Bjoern Oleson, Thomas Schmidt, Leila Taher, Dion Whitehead, Jürgen Borlak (ed.). *John Wiley & Sons* (2005).<br />
- [Evaluating an Evolutionary Approach for Reconstructing Gene Regulatory Networks.](content/courses/alife9.pdf) Whitehead, D. and Skusa, A. and Kennedy, P.J. *Ninth International Conference on the Simulation and<br />
Synthesis of Living Systems* (2004).</p>
